opsdroid/configuration/schema.yaml

```diff
@@ -0,0 +1,118 @@
+welcome-message: bool(required=False)
+
+connectors:
```

Comment:
At least one of these connectors is required otherwise opsdroid will not work, basically if no connectors are found opsdroid will raise a `Critical` logging saying that all connectors failed to load and the application will exit.

This is what happens when you run opsdroid without any connector configured:
```python
CRITICAL opsdroid.core: No connectors in configuration, at least 1 required
INFO opsdroid.core: Exiting application with return code 1
```

I'm going to say that at least the websocket connector should be required in the configuration.

Reply:
This connector scheme: the list (include ('websocket, required = False), .., include (' twitter ', required = False)) requires at least one connector. Added test to check file without connectors: test_load_config_broken_without_connectors. 
 If this scheme looked like this: 
  connectors: list(include('websocket', required=False),...,include('twitter',required=False),required=False) the presence of connectors would not be necessary

Reply:
Thats great thanks for the explanation 😄 👍